nvflare/app_opt/pt/params_converter.py

```diff
@@ -14,16 +14,22 @@
 
 from typing import Dict
 
+import numpy as np
 import torch
 
 from nvflare.app_common.abstract.params_converter import ParamsConverter
 
 
 class NumpyToPTParamsConverter(ParamsConverter):
     def convert(self, params: Dict, fl_ctx) -> Dict:
-        return {k: torch.as_tensor(v) for k, v in params.items()}
+        tensor_shapes = fl_ctx.get_prop("tensor_shape_dict")
+        if tensor_shapes:
+            return {k: torch.as_tensor(np.reshape(v, tensor_shapes[k])) for k, v in params.items()}
```

Comment:
should we only reshape if `k` is in ` tensor_shapes`?

Reply:
sure, good idea